### PR TITLE
Another tsc run pass which found some invalid properties

### PR DESCRIPTION
**Pull request by jelly**

### CODE REVIEW

**Comment by jelly** [comment]
especially fun `cockpit.warn` :)

**Review by allisonkarlitskaya — COMMENTED** [review]
`pkg/networkmanager/interfaces.js` has another instance of `cockpit.warn()`, probably worth fixing too

**Comment by jelly** [comment]
> `pkg/networkmanager/interfaces.js` has another instance of `cockpit.warn()`, probably worth fixing too

yes! Nicely spotted!

File: pkg/networkmanager/interfaces.js [hunk 1]
```diff
@@ -1455,7 +1455,7 @@ export function settings_applier(model, device, connection) {
         } else if (device) {
             return device.activate_with_settings(settings);
         } else {
-            cockpit.warn("No way to apply settings", connection, settings);
+            console.warn("No way to apply settings", connection, settings);
```
Comment (cockpituous) [inline pkg/networkmanager/interfaces.js:1458]: This added line is not executed by any test.

File: pkg/lib/cockpit-components-password.jsx [hunk 2]
```diff
@@ -123,7 +123,6 @@ export const PasswordFormFields = ({
                                </button>
                            </Popover>
                        }
-                       validated={error_password ? "warning" : "default"}
```
Comment (allisonkarlitskaya) [inline pkg/lib/cockpit-components-password.jsx:126]: I wonder what the intention was here.  This kinda looks like a `variant` field, except this component doesn't have one.

In any case, this doesn't make anything worse.

**Review by allisonkarlitskaya — APPROVED** [review]

**Comment by martinpitt** [comment]
Ugh, lots and lots of failures, but they all look unsystematic/flakes. Retrying once.

File: pkg/lib/cockpit-components-password.jsx (hunk 2 above)
Comment (martinpitt) [reply pkg/lib/cockpit-components-password.jsx:126]: The `validated` property normally goes into the input element *inside* of the FormGroup, like e.g. [here in kdump](https://github.com/cockpit-project/cockpit/blob/294fd5305c9eefc41cff4d59a06f78a0040c6843/pkg/kdump/kdump-view.jsx#L299). Firewall and lots of other places use that, too.

In this case it was meant to go into the `TextInput className="check-passwords" ` below, but that already has it. So removing it here is correct.

**Review by martinpitt — APPROVED** [review]
Cheers!

File: pkg/networkmanager/interfaces.js (hunk 1 above)
Comment (cockpituous) [inline pkg/networkmanager/interfaces.js:1458]: This added line is not executed by any test.

**Pull request merged**